sdk/appcenter-auth/src/test/java/com/microsoft/appcenter/auth/AuthTest.java

```diff
@@ -284,17 +289,41 @@ private void testInvalidConfig(JSONObject jsonConfig) throws Exception {
         Auth auth = Auth.getInstance();
         start(auth);
 
+        /* mock public client application*/
+        mockMsalPublicClientApplication();
+
         /* When we get a payload valid for AppCenter fields but invalid for msal ones. */
         mockSuccessfulHttpCall(jsonConfig, mHttpClient);
 
-        /* We didn't attempt to even save. */
+        /* We saved after we downloaded the file. */
+        verifyStatic();
+        FileManager.write(any(File.class), anyString());
+
+        /* We cleared the cache since the file was invalid. */
+        verifyStatic();
+        FileManager.delete(any(File.class));
```

Comment:
We return a valid msal public client app, so why do we delete the file given we don't simulate an exception? There might be something failing in the test for another reason.